src/lib.rs

```diff
@@ -219,23 +215,23 @@ where
     /// * `connector` - TLS connector for [`Client`][hyper::client::Client]
     pub fn new(connector: C) -> Self {
         Self {
-            capabalities: None,
+            capabilities: None,
             connector,
         }
     }
     /// Set chosen webdriver capabilities
-    pub fn capabilities(mut self, cap: webdriver::capabilities::Capabilities) -> Self {
-        self.capabalities = Some(cap);
+    pub fn capabilities(&mut self, cap: webdriver::capabilities::Capabilities) -> &mut Self {
+        self.capabilities = Some(cap);
         self
     }
     /// Connect to the webdriver session
     /// # Arguments
     /// * `webdriver` - webdriver url
-    pub async fn connect(self, webdriver: &str) -> Result<Client<C>, error::NewSessionError> {
-        return if let Some(cap) = self.capabalities {
-            Client::with_capabilities_and_connector(webdriver, cap, self.connector).await
+    pub async fn connect(&self, webdriver: &str) -> Result<Client<C>, error::NewSessionError> {
+        if let Some(cap) = self.capabilities.clone() {
```

Comment:
You shouldn't need the `.clone()` here I don't think — you just need to write either:
```rust
if let Some(ref cap) = self.capabilities {
```
or
```rust
if let Some(cap) = &self.capabilities {
```